src/senselab/audio/tasks/voice_cloning/api.py

```diff
@@ -0,0 +1,43 @@
+"""This module implements some utilities for the voice cloning task."""
+
+from typing import Any, Dict, List, Optional
+
+import pydra
+
+from senselab.audio.data_structures.audio import Audio
+from senselab.utils.data_structures.device import DeviceType
+from senselab.utils.data_structures.model import SenselabModel, TorchModel
+
+from .knnvc import KNNVC
```

Comment:
not sure if it would be better to use the full import path like above

Reply:
fixed